addons/diagnostic/fnc_initTargetDebugConsole.sqf

```diff
@@ -13,7 +13,10 @@ Author:
     commy2
 ---------------------------------------------------------------------------- */
 
-#define COUNT_WATCH_BOXES 8
+#define COUNT_WATCH_BOXES   8
+#define EXEC_RESULT         ([nil] apply {[] call _this} param [0, text ""])
```

Comment:
The watch fields report #NIL for nil return value and I like that you can see that something happened instead of void.

Vanilla reports:
```
nil -> 
[nil] -> [<any>]
```
So it only works for array elements there, but we can be better than that.

I'd wish for `text "<any>"` as default value, and also the same for the watch fields (could be part of this PR).

Reply:
Also, don't do this multiple spaces inside a line formatting. Should there ever be another macro added with a longer name, all the other spacing will have to be adjusted resulting in a pink wall. Don't do that.

Reply:
Suggestions applied.